server/channels/app/bot.go

```diff
@@ -458,11 +458,12 @@ func (a *App) disableUserBots(rctx request.CTX, userID string) *model.AppError {
 	perPage := 20
 	for {
 		options := &model.BotGetOptions{
-			OwnerId:        userID,
-			IncludeDeleted: false,
-			OnlyOrphaned:   false,
-			Page:           0,
-			PerPage:        perPage,
+			OwnerId:          userID,
+			IncludeDeleted:   false,
+			OnlyOrphaned:     false,
+			Page:             0,
+			PerPage:          perPage,
+			ExcludeSystemBot: true,
```

Comment:
I'm not against this approach, but I wonder if you checked if we can remove the `OwnerID` field from the system bot and solve the issue that way.

Reply:
I've checked and it should work. However by doing that we will have system-bot bot accounts hanging without ownerid 's. I think it will be hard to track back of the origins of these bots in the future.

Therefore I went for this approach.

Reply:
Fair, sounds good to me.